calicoctl/resourcemgr/ippool.go

```diff
@@ -29,35 +29,32 @@ func init() {
 		api.NewIPPoolList(),
 		false,
 		[]string{"ippool", "ippools", "ipp", "ipps", "pool", "pools"},
-		[]string{"CIDR"},
-		[]string{"CIDR", "NAT", "IPIP"},
+		[]string{"NAME", "CIDR"},
+		[]string{"NAME", "CIDR", "NAT", "IPIP", "DISABLED"},
 		map[string]string{
-			"CIDR": "{{.Metadata.CIDR}}",
-			"NAT":  "{{.Spec.NATOutgoing}}",
-			"IPIP": "{{if .Spec.IPIP}}{{.Spec.IPIP.Enabled}}{{else}}false{{end}}",
+			"NAME":     "{{.ObjectMeta.Name}}",
+			"CIDR":     "{{.Spec.CIDR}}",
+			"NAT":      "{{.Spec.NATOutgoing}}",
+			"IPIP":     "{{.Spec.IPIP.Mode}}",
```

Comment:
IPIP is still an optional struct so probably need to carry across the check that the struct exists, defaulting this value to "Always" if it doesn't exist.

I wonder if we should always set the Mode to Always in libcalico-go, so that querying the resource always returns this with a value?  WDYT?

Reply:
I think I like setting it in libcalico better because it's explicit. It's not always obvious what the default value is, so libcalico filling that in would be nice. I think then same _might_ go with BGPPeer, node field, we can set it to `(global)` if it;s not specified.

Reply:
I'm up for the IPIP one.  The Node one - displaying a value of (global) is really just a calicoctl nicety., so I don't think that one should.